### ISSUE TITLE
fork of which version of rescript?

### ISSUE THREAD

**Issue by fccm** [melange-re/melange#226]
It's not clear which version of ReScript the fork was made from.

I think it would be worth to know.
I'm using rescript's bsc 8.4.2. Did melange forked an earlier or later version of the rescript compiler?

**Comment by vphantom** [comment]
The fork happened around March 2021, if that helps. That said, the long-term plan is to incorporate updates from upstream so it might become difficult to compare the two after a while.

**Comment by anmonteiro** [comment]
It's a bit hard to say. I forked BuckleScript around August 2020 (check the first ever PR: https://github.com/melange-re/melange/pull/1), and then kept to date with upstream patches, with the general **exception** of:

- removing the Format, classes, other APIs
- optional keys in records 
- async / await
- JSXv4

So the runtime and APIs are kept fairly up to date, but we don't generally support idioms that are incompatible with OCaml. 

In the meantime, we've recently started keeping a fairly detailed [changelog](https://github.com/melange-re/melange/blob/main/Changes.md), which should help make it more obvious what's going on under the hood.
